Validator/Constraints/IsDifferentPasswordValidator.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace EMS\CoreBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class IsDifferentPasswordValidator extends ConstraintValidator
+{
+    const ERROR_MESSAGE = 'Password has to be different from the previous one';
```

Comment:
use the message from the constraint ?

Reply:
PHPStan does not allow this, I tried $constraint->message but I get the error "Access to an undefined property Symfony\Component\Validator\Constraint::$message." Although the code works.